reference/metadata_tables.rst

```diff
@@ -167,3 +173,65 @@ Citus manages shard health on a per-placement basis and automatically marks a pl
 |                |                      | | subsequent background activity.                                         |
 +----------------+----------------------+---------------------------------------------------------------------------+
 
+
+Worker Nodes table
+---------------------------------------
+
+The pg_dist_node table contains information about the worker nodes in the cluster. 
+
++----------------+----------------------+---------------------------------------------------------------------------+
+|      Name      |         Type         |       Description                                                         |
++================+======================+===========================================================================+
+| nodeid         |         int          | | Auto-generated identifier for an individual node.                       |
++----------------+----------------------+---------------------------------------------------------------------------+
+| groupid        |         int          | | Identifier used to denote a group of 1 primary and 0 or more            |
```

Comment:
Hesitant to mark as 'standby', since that name implies disuse. However in MX, these servers can take requests. Changing to primary server and secondary servers.